src/Utils/Create/Users.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Console\Utils\Create\Users.
+ */
+
+namespace Drupal\Console\Utils\Create;
+
+use Drupal\Console\Utils\Create\Base;
+use Drupal\Component\Utility\Unicode;
+use Drupal\Core\Entity\EntityManagerInterface;
+use Drupal\Core\Datetime\DateFormatterInterface;
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\user\Entity\Role;
+
+/**
+ * Class Users
+ * @package Drupal\Console\Utils
+ */
+class Users extends Base
+{
+    /**
+     * Vocabularies constructor.
+     *
+     * @param EntityManagerInterface $entityManager
+     * @param DateFormatterInterface $dateFormatter
+     */
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        DateFormatterInterface $dateFormatter
+    ) {
+        parent::__construct($entityManager, $dateFormatter);
+    }
+
+    /**
+     * Create and returns an array of new Vocabularies.
```

Comment:
`Create and returns an array of new Vocabularies.`
must read as 
`Create and returns an array of new Users.`